desktop/libs/notebook/src/notebook/connectors/sql_alchemy.py

```diff
@@ -296,7 +296,10 @@ def check_status(self, notebook, snippet):
     response = {'status': 'canceled'}
 
     if connection:
-      if snippet['result']['handle']['has_result_set']:
+      cursor = connection['result'].cursor
+      if self.options['url'].startswith('presto://') and cursor and cursor.poll():
```

Comment:
For the record using the dialect should have worked for sure with connectors, but at least this will be good with Editor v1

Reply:
It's consider for two presto cluster. The second one's dialect cannot be presto?

Reply:
I see, makes sense. 
The good news is that it will be soon easier to have multiple configured interpreters of the same dialect with the connectors: https://github.com/cloudera/hue/issues/1491